sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/NativeMethodsShouldBeWrapped.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+
+namespace Tests.Diagnostics

```

Comment:
Please add sample with invalid code.

Reply:
Other test cases:
* _internal_ and _protected_ 
* public but effective scope is not public